modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -81,7 +82,7 @@ exports.signin = function (req, res, next) {
 
   passport.authenticate('local', function (err, user, info) {
     if (err || !user) {
-      res.status(400).send(info);
+      res.status(401).send(info);
```

Comment:
Looks like some other code is still getting in here somehow

Reply:
nope intentional patch for authenitcation failure.  400 is bad request -- 401 is way more obvious. Came across this while debugging some login issues

Reply:
Ah okay, thats fair.